presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -65,11 +65,32 @@ Date and Time Functions
 
 .. function:: from_iso8601_timestamp(string) -> timestamp with time zone
 
-    Parses the ISO 8601 formatted ``string`` into a ``timestamp with time zone``.
+    Parses the ISO 8601 formatted date and time ``string``, optionally with
+    time and timezone, into a ``timestamp with time zone``. Missing
+    information is defaulted to zero and local information::
+
+        SELECT from_iso8601_timestamp('2020-05-11');
+        2020-05-11 00:00:00.000 America/Vancouver
+
+        SELECT from_iso8601_timestamp('2020-05-11T11:15:05');
+        2020-05-11 11:15:05.000 America/Vancouver
+
+        SELECT from_iso8601_timestamp('2020-05-11T11:15:05.055+01:00');
+        2020-05-11 11:15:05.055 +01:00
 
 .. function:: from_iso8601_date(string) -> date
 
-    Parses the ISO 8601 formatted ``string`` into a ``date``.
+    Parses the ISO 8601 formatted date ``string`` into a ``date``. Various date
```

Comment:
Instead of saying "various", which implies these are just some of the supported formats, let's change to enumerate them, using the language from standard (or at least what Wikipedia calls them):

> The date can be a calendar date, a week date using ISO week numbering, or an ordinal date (year and day of year).